src/Ocelot/DownstreamRouteFinder/HeaderMatcher/IHeaderPlaceholderNameAndValueFinder.cs

```diff
@@ -0,0 +1,12 @@
+﻿using Ocelot.DownstreamRouteFinder.UrlMatcher;
+using Ocelot.Values;
+
+namespace Ocelot.DownstreamRouteFinder.HeaderMatcher;
+
+/// <summary>
+/// Ocelot feature: <see href="https://github.com/ThreeMammals/Ocelot/blob/develop/docs/features/routing.rst#upstream-headers">Routing based on request header</see>.
+/// </summary>
+public interface IHeaderPlaceholderNameAndValueFinder
+{
+    List<PlaceholderNameAndValue> Find(Dictionary<string, string> upstreamHeaders, Dictionary<string, UpstreamHeaderTemplate> templateHeaders);
```

Comment:
```suggestion
    IList<PlaceholderNameAndValue> Find(Dictionary<string, string> upstreamHeaders, Dictionary<string, UpstreamHeaderTemplate> templateHeaders);
```

Reply:
Fixed in commit https://github.com/ThreeMammals/Ocelot/pull/1312/commits/c727b934719503710b60671f22b0db25c05207e4 ✔️